test/metabase/search/filter_test.clj

```diff
@@ -76,26 +78,63 @@
             [:= :card.archived false]]
            (:where (search.filter/build-filters
                     base-search-query "card"
-                    (merge default-search-ctx {:search-string "a string"}))))))
-
+                    (merge default-search-ctx {:search-string "a string"})))))))
+
+(deftest  build-created-at-filter-test
```

Comment:
this is the added test, the rests are the just breaking tests to smaller deftest